### DISCUSSION TITLE
Sections - Any way to fill the game that is circled?

### DISCUSSION THREAD

**Discussion by ShadowIsADev** [home-assistant/frontend#23494] [Dashboard]
Is there anyway to move the sections that are under that huge gap up so it looks a lot better?![Screenshot 2024-12-29 3 02 37 AM](https://github.com/user-attachments/assets/232f568d-b8a5-46ef-a420-4e19dca59696)

**Comment by emufan** [comment]
Yes. Put a 

```
row_span: 2
```

in the left section. Not in the card, but in the section via three-dot-yaml

n=2 is only example here. I use n+1 most of the times to get a better look.

![image](https://github.com/user-attachments/assets/e7cf23ee-0d4a-49c5-8ae3-206e7a9ed5c0)
